doc/classes/AudioEffectAmplify.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectAmplify" inherits="AudioEffect" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Amplifies the volume of an audio source.
+		Adds a Amplify audio effect to an Audio bus.
+		Increases or decreases the volume of the audio bus you have selected
```

Comment:
Missing period. Also, try: "volume of the selected audio bus." or "the selected audio bus' volume."